### DISCUSSION TITLE
Can't open XDM

### DISCUSSION THREAD

**Discussion by Marvin1932** [subhra74/xdm#544] [Q&A]
How to fix 'an older version of XDM is already running' problem. I'm using windows 10.

**Comment by sp00kie** [comment]
same problem. restart does nothing. windows 10

**Comment by subhra74** [comment]
Please uninstall all previous version of XDM. also delete XDM folder from program files and re install latest version

**Reply by Marvin1932** [reply]
I have removed all versions of XDM and I deleted the XDM file in Program Files. I then reinstalled XDM 2018 and I still get the same message - "An older version of XDM is already running". Hopefully you have some other suggestions. Thank you.

Marv Beer

**Comment by Marvin1932** [comment]
If I haven't replied before I just have tried that and it has not helped. I'm completely stumped.

Marv Beer

On June 7, 2021, at 2:13 AM, Subhra Das Gupta ***@***.***> wrote:

Please uninstall all previous version of XDM. also delete XDM folder from program files and re install latest version

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.￼

[ { ***@***.***": "<a href="http://schema.org">http://schema.org</a>", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "<a href="https://github.com/subhra74/xdm/discussions/544#discussioncomment-833456">https://github.com/subhra74/xdm/discussions/544#discussioncomment-833456</a>", "url": "<a href="https://github.com/subhra74/xdm/discussions/544#discussioncomment-833456">https://github.com/subhra74/xdm/discussions/544#discussioncomment-833456</a>", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "<a href="https://github.com">https://github.com</a>" } } ]

**Reply by Marvin1932** [reply]
I used ‘Advanced Systems Care’ to uninstall this program. The result was that XDM was removed from Program Files and %USERPROFILE%/.xdman was no longer found in the Register. I then restarted the computer and installed the program using your url listed below. I got the same result -  an older version is still running

I am at a complete loss as to where to go next.

**Comment by subhra74** [comment]
@Marvin1932 Please use latest version 7.2.11 from here https://github.com/subhra74/xdm/releases/download/7.2.11/xdm-setup.msi

Also when you uninstall and remove following folders:
1. XDM installation folder in Program files
2. XDM configuration folder in %USERPROFILE%/.xdman (this will remove all downloads in XDM)
3. Restart and install XDM 2020 ( 7.2.11)

**Comment by Marvin1932** [comment]
Hello,

I used ‘Advanced Systems Care’ to uninstall this program. The result was that XDM was removed from Program Files and %USERPROFILE%/.xdman was no longer found in the Register. I then restarted the computer and installed the program using your url listed below. I got the same result - 

 

I am at a complete loss as to where to go next.

 

Marvin Beer

**Comment by Marvin1932** [comment]
Additional info.

I originally had xdm2016 installed and working. It was when I went to a newer version that my trouble began. Should I reinstall (if I had it) version 2016?

 

Also, are the newer versions listed in the Task Manager, and if so, where?

 

Thanks for trying to help!

 

Marvin Beer

**Comment by subhra74** [comment]
@Marvin1932 
I believe step 2 was not performed correctly. So I would explain a bit more here.
First Go to file explorer and navigate to` C:\Program Files\XDM` or` C:\Program Files (x86)\XDM `depending on 64 or 32 bit Windows. If XDM folder exists delete it. In case you get any errors like file can not be deleted - kill the XDM process (java.exe) using task manager and try deleting again.
Then paste this address `%USERPROFILE%/.xdman` in file explorer navigation bar to switch the folder, which is usually` C:\Users\<your-user-name>\.xdman` and delete the folder. Then install 7.2.11 again, this should fix the issue. Also use control panel to uninstall the app

**Comment by Marvin1932** [comment]
I want to thank you so much. I no longer am getting that ‘ another version is already running’ message. Now it is a case of me learning how to run the program. 

 

Thank you again for all your help.

 

Marvin Beer

**Reply by vehosah** [reply]
i did all of this,i installed the new version ,i click on it and it nothing opens

**Reply by WangJH95** [reply]
I have the same problem in both linux and windows systems, have you solve it?

**Reply by vehosah** [reply]
> I have the same problem in both linux and windows systems, have you solve it?

No I didn't solve it
